packages/core-components/src/components/WarningPanel/WarningPanel.test.tsx

```diff
@@ -17,22 +17,39 @@
 import React from 'react';
 import { fireEvent, screen } from '@testing-library/react';
 import { renderInTestApp } from '@backstage/test-utils';
-import { Typography } from '@material-ui/core';
+import { ThemeProvider, Typography } from '@material-ui/core';
 
-import { WarningPanel } from './WarningPanel';
+import { WarningPanel, WarningProps } from './WarningPanel';
+import { lightTheme } from '@backstage/theme';
 
-const propsTitle = { title: 'Mock title' };
-const propsTitleMessage = { title: 'Mock title', message: 'Some more info' };
-const propsMessage = { message: 'Some more info' };
+const propsTitle: WarningProps = { title: 'Mock title' };
+const propsTitleMessage: WarningProps = {
+  title: 'Mock title',
+  message: 'Some more info',
+};
+const propsMessage: WarningProps = { message: 'Some more info' };
+const propsErrorMessage: WarningProps = {
+  severity: 'error',
+  title: 'Mock title',
+  message: 'Some more info',
+};
 
 describe('<WarningPanel />', () => {
   it('renders without exploding', async () => {
-    await renderInTestApp(<WarningPanel {...propsTitleMessage} />);
+    await renderInTestApp(
```

Comment:
`renderInTestApp` provides the theme, so confused why `ThemeProvider` is needed here 🤔